makefiles/pseudomodules.inc.mk

```diff
@@ -59,6 +59,7 @@ PSEUDOMODULES += gnrc_sixlowpan_border_router_default
 PSEUDOMODULES += gnrc_sixlowpan_default
 PSEUDOMODULES += gnrc_sixlowpan_frag_hint
 PSEUDOMODULES += gnrc_sixlowpan_frag_sfr_stats
+PSEUDOMODULES += gnrc_sixlowpan_frag_sfr_congure
```

Comment:
Should we have a 

```make
ifneq (,$(filter gnrc_sixlowpan_frag_sfr_congure,$(USEMODULE)))
  USEMODULE += gnrc_sixlowpan_frag_sfr
endif
```

Reply:
You mean like https://github.com/RIOT-OS/RIOT/pull/16156/files#diff-6e381f8f6f5081ba31d65f4cd98d2d96bc6f35e97fb33513fe8453ca71cd9834R247-R249?